code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -32,3 +32,5 @@
 	var/tinttotal = 0	// Total level of visualy impairing items
 
 	var/list/bodyparts = list() //Gets filled up in the constructor (New() proc in human.dm and monkey.dm)
+
+	var/obj/effect/proc_holder/ranged_ability //Any ranged ability the mob has, as a click override

```

Comment:
...Why do you have to do this to me? I will _try_ to move this up to /living if it does not cause too much trouble.